scripts/classes/card.gd

```diff
@@ -17,28 +33,20 @@ func _on_area_2d_input_event(viewport, event, shape_idx):
 	# other possible event type here: InputEventMouseMotion
 	# can be used to track position of the mouse inside the card
 
-func update_face():
-	$Image.texture = card_data.image;
-
 ## Activates when mouse button is pressed
 func _on_click():
 	pass
 
-var opened = false
-
 ## Activates when mouse button is released (if still on card)
 func _on_click_confirm():
 	clicked.emit(self)
-	if opened:
-		$AnimationPlayer.play("card_close")
-	else:
-		$AnimationPlayer.play("card_open")
-	opened = !opened
-	
-
 
 func _on_area_2d_mouse_entered():
 	pass
 
 func _on_area_2d_mouse_exited():
 	pass
+
+func update_face():
```

Comment:
Since we also update the back of the card, might want to rename this.
update_visual() or load_card_data()

Reply:
Agreed, The name has been updated